project/Build.scala

```diff
@@ -32,7 +32,7 @@ object SlickBuild extends Build {
     val reactiveStreamsVersion = "1.0.0"
     val reactiveStreams = "org.reactivestreams" % "reactive-streams" % reactiveStreamsVersion
     val reactiveStreamsTCK = "org.reactivestreams" % "reactive-streams-tck" % reactiveStreamsVersion
-    val hikariCP = "com.zaxxer" % "HikariCP-java6" % "2.3.13"
+    val hikariCP = "com.zaxxer" % "HikariCP" % "2.5.1"
```

Comment:
According to http://slick.lightbend.com/news/2016/12/05/slick-3.2.0-M2-released.html:
* "From this release on, Slick requires Java 8. Older JVMs are no longer supported."

So should be fine for 3.2+?